### PR TITLE
Feature/create user covers

### PR DESCRIPTION
**Pull request by ADavidson02**
## Is this a fix or a feature?
feature

## What is the change?
completes iteration 3, completes functionality of "make my book" button.
-declare a new instantiation of the Cover class with the users input.
-pushed the users input to the data.js arrays.
-when user clicks make my book button, they are taken back to the home view, 
where their book cover is being displayed

## What does it fix?
n/a

## Where should the reviewer start?
-line 17 of main.js

## How should this be tested?
-from home view click make your own cover button
- fill in input fields * use url for cover field
-click make my book

#### Screenshots (if appropriate)

Screen Shot 2020-08-29 at 12 52 41 PM

Screen Shot 2020-08-29 at 12 52 56 PM

@ADavidson02,
@saraho1123

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -14,9 +14,17 @@ var coverTagLine2 = document.querySelector(".tagline-2");
 
 var showRandomButton = document.querySelector(".random-cover-button");
 var homeButton = document.querySelector(".home-button");
-var makeNewButton = document.querySelector(".make-new-button");
+var makeYourCoverButton = document.querySelector(".make-new-button");
```
Comment (saraho1123) [inline src/main.js:17]: I like the change of name for this variable! It makes much more sense.

File: src/main.js [hunk 2]
```diff
@@ -14,9 +14,17 @@ var coverTagLine2 = document.querySelector(".tagline-2");
...
 var viewCoversButton = document.querySelector(".view-saved-button");
 var saveCoverButton = document.querySelector(".save-cover-button");
+var myBookButton = document.querySelector(".create-new-book-button");
+
+var userCoverInput = document.querySelector(".user-cover");
+var userTitleInput = document.querySelector(".user-title");
+var userDescriptionInput1 = document.querySelector(".user-desc1");
+var userDescriptionInput2 = document.querySelector(".user-desc2");
+
```
Comment (saraho1123) [inline src/main.js:26]: We should clean up our spacing a bit.

File: src/main.js [hunk 3]
```diff
@@ -77,10 +87,38 @@ function displayHomeView() {
...
+  var userBookCover = new Cover (
+    userCoverInput.value,
+    userTitleInput.value,
+    userDescriptionInput1.value,
+    userDescriptionInput2.value,
+  )
+  displayCover(userBookCover);
+}
+
+//declare a variable inside of a function and
```
Comment (saraho1123) [inline src/main.js]: We should clean up our pseudocode. So glad we are remembering to pseudocode and console log!

**Review by saraho1123 — CHANGES_REQUESTED** [review]
Let's clean it up before we merge to main! We are doing awesome!

**Comment by ADavidson02** [comment]
Cleanup has been completed.

**Comment by saraho1123** [comment]
Looks great! I know we agreed to leave spaces in our query selectors for now, until we do the final PR before we turn in the project. I am very happy with our agreement about that! :)

**Pull request merged**